kissnet.hpp

```diff
@@ -1005,7 +1005,12 @@ namespace kissnet
 				if (sock == INVALID_SOCKET)
 					kissnet_fatal_error("unable to create connectable socket!");
 
-				auto* pMethod = TLSv1_2_client_method();
+                auto* pMethod =
```

Comment:
indentation looks wrong.

Reply:
nothing clang-format will not fix next time it's ran